apps/blockchain/lib/blockchain/transaction.ex

```diff
@@ -439,6 +455,10 @@ defmodule Blockchain.Transaction do
     end
   end
 
+  defp create_receipt(state_root_hash, gas_used, logs, _status_code) do
```

Comment:
We're not using the `status_code` here, should we be using it? If I recall correctly, the `state_root` needed to change based on whether or not the transaction succeeded or failed.

Reply:
yes, it's not used yet. I was gonna add in the next PRs.